docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,154 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives stream of POJOs as input and using JDBC connection will insert the POJOs in a database table where each POJO corresponds to a row in the database table. The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. To execute/fire the insert queries on the database table, this operator uses PreparedStatement, a precompiled SQL statement.
+This operator internally extends the functionality of AbstractPassThruTransactionableStoreOutputOperator.This is the base implementation for a pass-through output adapter of a transactional store.This implementation provides the "exactly once" guarantee."Pass-through" means it does not wait for end window to write to the store. It will begin transaction at begin window and write to the store as the tuples come and commit the transaction at end window.
+
+**Note**: For enforcing exactly once semantics a table named dt_meta must exist in the database. The SQL to create the same is as follows
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))"
```

Comment:
@tweise changing the table may cause a problem for the applications already using it as it is hard coded.
If you still want it to change let me know I will open a separate Jira and do the needful.